knee_sarg/oai/run_triggers.py

```diff
@@ -0,0 +1,92 @@
+from pydantic import Field
+from pathlib import Path
+from dagster import (
+    asset,
+    Config,
+    AssetKey,
+    EventLogEntry,
+    SensorResult,
+    RunRequest,
+    SensorEvaluationContext,
+    asset_sensor,
+    DefaultSensorStatus,
+)
+from dagster_duckdb import DuckDBResource
+import os
+import json
+import polars as pl
+import pandas as pd
+
+from ..resources import (
+    DATA_DIR,
+)
+from ..ingest.ingested_study import study_uid_partitions_def
+from ..jobs import (
+    cartilage_thickness_oai_job,
+)
+
+
+class StudyUidConfig(Config):
+    study_uid_file: str = Field(
+        default_factory=lambda: "study_uids_to_run.json",
+        description="JSON file with array of study UIDs",
+    )
+
+
+@asset()
+def oai_study_uids_to_run(
```

Comment:
Materialziing this asset will start `cartilage_thickness_oai_job`s for each study UID in `oai-sampler/study_uids_to_run.json`